src/main/java/de/fraunhofer/aisec/cpg/passes/scopes/RecordScope.java

```diff
@@ -30,6 +30,8 @@
 
 public class RecordScope extends NameScope {
 
+  enum Visiblity {}
```

Comment:
What exactly does an enum without entries do :sweat_smile:

Reply:
Leftover... I wanted to start implementing visibility of class members but decided against it. I can remove it.